csvcubed/csvcubed/cli/inspect/metadataprocessor.py

```diff
@@ -0,0 +1,33 @@
+"""
+Metadata Processor
+---------------------------
+
+Provides functionality for validating and detecting input metadata.json file.
+"""
+
+import logging
+
+from pathlib import Path
+from csvcubed.cli.inspect.metadatainputhandler import MetadataType
+from rdflib import Graph
+
+_logger = logging.getLogger(__name__)
+
+
+class MetadataProcessor:
+    """
+    This class handles the conversion/serialisation of metadata jsons to RDF.
+    """
+
+    def serialise_to_rdf(self, metadata_json: Path, type: MetadataType) -> Graph:
```

Comment:
I'm not sure `serialise` is the right word here, I'd say `load_json_ld_to_rdflib_graph` or something like that.

Reply:
@robons I agree, I will update accordingly.